auction-app-backend/src/main/java/com/example/auctionapp/entity/UserEntity.java

```diff
@@ -50,6 +53,16 @@ public class UserEntity implements UserDetails {
     @JoinColumn(name = "payment_info_id")
     private PaymentInfoEntity paymentInfoEntity;
 
+    @Column(name = "profile_picture")
+    private String profilePicture;
+
+    @Column(name = "dob")
+    @Temporal(TemporalType.DATE)
+    private LocalDate dob;
```

Comment:
I guess this is date of birth, so please avoid abbreviations. Instead use full name date_of_birth